kollect-core/src/main/kotlin/kollect/arrow/instances/core/instances.kt

```diff
@@ -1,16 +1,19 @@
 package arrow.core
 
 import arrow.Kind
-import arrow.typeclasses.MonadError
+import arrow.instance
+import arrow.instances.TryMonadInstance
 import kollect.KollectMonadError
 import kollect.Query
 
-object TryKollectMonadErrorInstance : KollectMonadError<ForTry> {
-    override fun ME(): MonadError<ForTry, Throwable> = Try.monadError()
+@instance(Try::class)
+interface TryKollectMonadErrorInstance : TryMonadInstance, KollectMonadError<ForTry> {
```

Comment:
TryMonadErrorInstance?

Reply:
@pakoito the issue here is that the Monad here is a Free monad and uses suspension so it required a monad error that was custom but once this code is moved to arrow effect this type class and instance won't be necessary